x-pack/filebeat/input/azureeventhub/config_test.go

```diff
@@ -55,3 +55,92 @@ func TestValidate(t *testing.T) {
 		)
 	})
 }
+
+func TestValidateConnectionStringV1(t *testing.T) {
+	t.Run("Connection string contains entity path", func(t *testing.T) {
+		config := defaultConfig()
+		config.ProcessorVersion = "v1"
+		config.ConnectionString = "Endpoint=sb://my-namespace.servicebus.windows.net/;SharedAccessKeyName=my-key;SharedAccessKey=my-secret;EntityPath=my-event-hub;"
+		config.EventHubName = "my-event-hub"
+		config.SAName = "teststorageaccount"
+		config.SAKey = "my-secret"
+		config.SAContainer = "filebeat-activitylogs-event_hub_00"
+
+		if err := config.Validate(); err != nil {
+			t.Fatalf("unexpected validation error: %v", err)
+		}
```

Comment:
[NIT]
As you're already using testify, you could instead use `require`

```suggestion
		err := config.Validate()
		require.NoError(t, err, "unexpected validation error)
		
```

Reply:
Great point, `require` is much better. Applying them all..